src/adapter/debugAdapter.ts

```diff
@@ -117,6 +117,11 @@ export class DebugAdapter implements IDisposable {
     this.dap.on('setExcludedCallers', params => this._onSetExcludedCallers(params));
     this.dap.on('saveDiagnosticLogs', ({ toFile }) => this._saveDiagnosticLogs(toFile));
     this.dap.on('setSourceMapStepping', params => this._setSourceMapStepping(params));
+    this.dap.on('setDebuggerProperty', params => this._setDebuggerProperty(params));
+  }
+
+  private _setDebuggerProperty(params: Dap.SetDebuggerPropertyParams) {
+    this._thread?.cdp().DotnetDebugger.setDebuggerProperty(params);
```

Comment:
Since the DAP request is generic, should this too have a more generic domain?

Reply:
Fixed.